config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root 'home#index'
+  resources :users, only: [:show] do
+    resources :posts, only: [:index, :show]
+  end
```

Comment:
- [ ] Great job on working through the routes! 🥇🥇 I noticed that you've set a root route to `home#index`, which is fantastic for defining the landing page of your application. However, there's an additional route mentioned in the project requirements that's currently missing. That's what I'm currently getting 👇

![image](https://github.com/isume295/Blog-app/assets/106119105/62ac9263-8559-40fb-b074-9e13f48091bf)

That's what we should have according to the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md)

![image](https://github.com/andrianarivo/blog_app/assets/106119105/e276c441-8606-4d91-97d3-e3453ab10821)

To achieve this, you can add a route for listing all users like this 👇

``` ruby
# This is your root route, which sets the default landing page of your application
root 'home#index'

# This route displays a list of all users, as per the project requirements
get '/users', to: 'users#index'

# or you can simply use the following for users
resources :users, only: [:index, :show]
```

Reply:
done